### PR TITLE
fix monaco-editor-webpack-plugin

### PR DESCRIPTION
**Pull request by domsew**
fix monaco-editor-webpack-plugin after library output change

### CODE REVIEW

File: examples/monaco-editor-webpack-plugin/webpack.config.js [hunk 1]
```diff
@@ -38,13 +32,14 @@ export default {
       customLanguages: [
         {
           label: 'yaml',
+          // eslint-disable-next-line prettier/prettier
```
Comment (remcohaszing) [inline examples/monaco-editor-webpack-plugin/webpack.config.js:35]: Can you remove this line and format the code?

File: examples/monaco-editor-webpack-plugin/webpack.config.js [hunk 2]
```diff
@@ -38,13 +32,14 @@ export default {
...
           entry: [
-            'monaco-yaml/lib/esm/monaco.contribution',
+            'monaco-yaml/index',
             'vs/basic-languages/yaml/yaml.contribution',
```
Comment (remcohaszing) [inline examples/monaco-editor-webpack-plugin/webpack.config.js:38]: Does it work if you change this to just `'monaco-yaml'`?

**Review by remcohaszing — CHANGES_REQUESTED** [review]
Thanks! Looks good, just 2 minor comments. Can you have a look at them?

File: examples/monaco-editor-webpack-plugin/webpack.config.js (hunk 2 above)
Comment (domsew) [reply examples/monaco-editor-webpack-plugin/webpack.config.js]: Yeah, it does :)

File: examples/monaco-editor-webpack-plugin/webpack.config.js (hunk 1 above)
Comment (domsew) [reply examples/monaco-editor-webpack-plugin/webpack.config.js]: Done

**Comment by remcohaszing** [comment]
Awesome! Thanks!

**Pull request merged**